packages/ui-components/src/model_file_icon.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { File } from "lucide-solid";
+import { For } from "solid-js";
+import type { Meta, StoryObj } from "storybook-solidjs-vite";
+
+import { ModelFileIcon, theoryToLetterMap } from "./model_file_icon";
+
+const meta = {
+    title: "Icons/_internal/ModelFileIcon",
+    component: ModelFileIcon,
+} satisfies Meta<typeof ModelFileIcon>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Summary: Story = {
+    render: () => <div style={{ display: "flex", gap: "8px", "align-items": "center" }}></div>,
+    tags: ["!autodocs", "!dev"],
+};
+
+export const AllIcons: Story = {
+    render: () => (
+        <>
+            <For each={Object.keys(theoryToLetterMap)}>
+                {(modelType) => (
+                    <div>
+                        <ModelFileIcon theory={modelType} />
+                        {modelType}
+                    </div>
+                )}
+            </For>
+            <div>
+                <ModelFileIcon theory="unknown" />
+                unknown
+            </div>
+        </>
+    ),
+};
+
+export const DifferentSizes: Story = {
+    render: () => (
+        <For each={Object.keys(theoryToLetterMap)}>
+            {(modelType) => (
+                <div style={{ display: "flex", gap: "16px", "align-items": "center" }}>
+                    <ModelFileIcon theory={modelType} size={16} />
+                    <ModelFileIcon theory={modelType} size={20} />
+                    <ModelFileIcon theory={modelType} size={24} />
+                    <ModelFileIcon theory={modelType} size={32} />
+                    <ModelFileIcon theory={modelType} size={48} />
+                </div>
+            )}
+        </For>
+    ),
+};
+
+export const DifferentColors: Story = {
+    render: () => (
+        <For each={["#000", "#fff", "#f00", "#0f0", "#00f"]}>
```

Comment:
I can't help but notice that the red, green, and blue colors here are very ugly. Not a priority right now but I figure at some point we should have a standard palette that we're using for the app? E.g., the alerts have more palatable choices of colors.